phpstan.dist.neon

```diff
@@ -13,6 +13,8 @@ parameters:
                 - src/Behat/Behat/Definition/Printer/ConsoleDefinitionPrinter.php
                 - src/Behat/Behat/EventDispatcher/Cli/StopOnFailureController.php
                 - src/Behat/Behat/Output/Node/Printer/ListPrinter.php
+                - src/Behat/Behat/Output/Node/Printer/Pretty/PrettyPathPrinter.php
```

Comment:
Now we are not using some constructor arguments in some classes but we can't remove them so we tell PhpStan to ignore them